pkgs/applications/radio/rscw/default.nix

```
@@ -0,0 +1,38 @@
+{ lib
+, stdenv
+, fetchurl
+, fftw
+, gtk2
+, pkg-config
+}:
+
+stdenv.mkDerivation rec {
+  pname = "rscw";
+  version = "0.1e";
+
+  src = fetchurl {
+    url = "https://www.pa3fwm.nl/software/${pname}/${pname}-${version}.tgz";
+    sha256 = "1hxwxmqc5jinr14ya1idigqigc8qhy1vimzcwy2vmwdjay2sqik2";
+  };
+
+  setSourceRoot = "sourceRoot=`pwd`";
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ gtk2 fftw ];
+
+  installPhase = ''
+    mkdir -pv $out/bin
+    install -m 0755 noisycw $out/bin/noisycw
```

```suggestion
    install -Dm 0755 noisycw $out/bin/noisycw
```
-D creates the directory for us